pyproject.toml

```diff
@@ -12,9 +12,9 @@ classifiers = [
     "Development Status :: 1 - Planning",
     "License :: OSI Approved :: GNU General Public License (GPL)",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.12",
 ]
-requires-python = ">=3.10"
+requires-python = ">=3.11"
```

Comment:
Are we sure 3.11 is the minimal, or should that be 3.12 here as well?

Reply:
I was also wondering. Because if we consider the tool is to be used only as a snap then we do not really care for previous python versions. I am personally in favor of dropping support for python 3.11 (less linter fights, less tests to run, etc.) but I may miss some contraints. What do you think about that?

Reply:
I'd be +1 on that. Doesn't have to be changed as part of this PR though.